webroot/results/admin/scripts/check_comp_data.php

```diff
@@ -150,27 +150,10 @@
 
 
 // table to check existence of results vs scrambles
-$eventIdResults = $wcadb_conn->boundQuery( "SELECT DISTINCT(eventId) FROM Results WHERE competitionId=?", array('s', &$compId));
-$eventIds = array_unique(array_map(function($eventIdResult) {
-  return $eventIdResult["eventId"];
-}, $eventIdResults));
-$eventOptionsStr = implode("", array_map(function($eventId) {
-  $selected = $eventId == "333" ? "selected" : "";
-  return "<option ${selected} value='${eventId}'>${eventId}</option>";
-}, $eventIds));
 print "<li><p>Sanity Checks:</p>
          <ol type='a'>
            <li><a href='/competitions/$compIdUrl' target='_blank' class='link-external external'>View the Public competition page</a></li>
-           <li>
-             Post the
-             <a data-competition-id='$compIdUrl' href='#' id='post-results-link' target='_blank' class='link-external external'>
-               results announcement
-             </a>
-             <select id='main-event-select'>
-               <option value=''>No main event</option>
-               ${eventOptionsStr}
-             </select>
-           </li>
+           <li><a href='/competitions/$compIdUrl/edit/admin' target='_blank' class='link-external external'>Post the results announcement</a></li>
```

Comment:
Side note: while looking on this I realized that now it's less "posting results announcement" and more like "sending results announcement" / "announcing results", but I don't feel strongly about changing that ^^